src/EditorFeatures/Test/Workspaces/ProjectCacheHostServiceFactoryTests.cs

```diff
@@ -303,6 +302,26 @@ public override TWorkspaceService GetService<TWorkspaceService>()
 
                 return default;
             }
+
+            private sealed class MockOptionService : ILegacyWorkspaceOptionService
```

Comment:
wha'ts the future of ILegacyWorkspaceOptionService?

Reply:
Will document. It's only needed to support obsolete public API. Can remove once we remove that API (Workspace.Options, Solution.Options).